src/commcare_cloud/commands/ansible/helpers.py

```diff
@@ -122,3 +120,54 @@ def run_action_with_check_mode(run_check, run_apply, skip_check, quiet=False, al
             puts(colored.red(u"✗ Apply failed with status code {}".format(exit_code)))
 
     return exit_code
+
+
+ProcessDescriptor = namedtuple('ProcessDescriptor', 'host, short_name, number, full_name')
+
+
+def get_celery_workers(environment):
+    """
+    A generator that yields ``ProcessDescriptor`` tuples for celery
+
+    The same process may be yielded more than once if a single process is managing
+    multiple queues.
+
+    :param environment:
+    """
+    for host, queues in environment.app_processes_config.celery_processes.items():
+        if not host or host == 'None':
+            continue
+        for comma_separated_queue_names, config in queues.items():
+            for queue in comma_separated_queue_names.split(','):
+                for worker_num in range(config.get('num_workers', 1)):
+                    process_name = get_celery_worker_name(environment, comma_separated_queue_names, worker_num)
+                    yield ProcessDescriptor(host, queue, worker_num, process_name)
+
+
+def get_celery_worker_name(environment, comma_separated_queue_name, worker_num):
+    environment_environment = environment.meta_config.deploy_env
+    project = environment.fab_settings_config.project
+    return "{project}-{environment}-celery_{comma_separated_queue_name}_{worker_num}".format(
+        project=project,
+        environment=environment_environment,
+        comma_separated_queue_name=comma_separated_queue_name,
+        worker_num=worker_num
+    )
+
+
+def get_pillowtop_processes(environment):
+    """
+    A generator that yields ``ProcessDescriptor`` tuples for pillowtop
+    :param environment:
+    """
+    for host, pillows in environment.app_processes_config.pillows.items():
+        for name, params in pillows.items():
+            start = params.get('start_process', 0)
+            num_processes = params.get('num_processes', 1)
+            for num_process in range(start, start + num_processes):
+                process_name = "commcare-hq-{deploy_env}-pillowtop-{pillow_name}-{num_process}".format(
+                    deploy_env=environment.meta_config.deploy_env,
+                    pillow_name=name,
+                    num_process=num_process
+                )
```

Comment:
There was some work in progress for moving pillowtop out of fab to ansible by Emord when the service management was being added. Adding pillowtop was avoided at that moment so that this naming convention for the process name is not at two different places. Like for celery `get_celery_worker_name` was added and was then made the single location for celery worker name.
I think we should consider having something similar for pillowtop as well so that any changes are reflected at both places.